includes/html/pages/addhost.inc.php

```diff
@@ -26,69 +26,69 @@
     $hostname = strip_tags($_POST['hostname']);
     if (! \LibreNMS\Util\Validate::hostname($hostname) && ! IP::isValid($hostname)) {
         print_error("Invalid hostname or IP: $hostname");
-    }
-
-    $new_device = new \App\Models\Device(['hostname' => $hostname]);
-
-    if (Auth::user()->hasGlobalRead()) {
-        // Settings common to SNMPv2 & v3
-        if ($_POST['port']) {
-            $new_device->port = strip_tags($_POST['port']);
-        }
+    } else {
```

Comment:
This is more an FYI but this breaks adding devices from snmpsimd now, before it would throw the invalid error but carry on. Now it stops because of the else and you can't even force add a device.

Reply:
How are you adding devices?  I use `lnms dev:simulate <snmprec_file_name>` which does not use the addhost page.

Separately, wouldn't the the snmpsim device have valid info?

Reply:
Just the normal way for the demo server so we have some test devices on there.

I wasn't aware of the simulate command, I can use that for testing and dev.